react/features/device-selection/DeviceSelectionPopup.js

```diff
@@ -223,7 +228,9 @@ export default class DeviceSelectionPopup {
             disableBlanketClickDismiss: true,
             setAudioInputDevice: this._setAudioInputDevice,
             setAudioOutputDevice: this._setAudioOutputDevice,
-            setVideoInputDevice: this._setVideoInputDevice
+            setVideoInputDevice: this._setVideoInputDevice,
+            onAudioPreviewUpdated: this._onAudioPreviewUpdated,
+            onVideoPreviewUpdated: this._onVideoPreviewUpdated
```

Comment:
Are _onAudioPreviewUpdated _onVideoPreviewUpdated defined for this component? I couldn't find them in this file.